README.md

```diff
@@ -160,7 +169,7 @@ Of the three following options, which is the most "correct" way of organizing th
 
 **Your answer:**
 
-> ...
+> B. Just looks good to me.
```

Comment:
:+1: because related components (controllers and their views) are right next to each other.